packages/default/scss/icons/_layout.scss

```diff
@@ -283,6 +283,7 @@
     .k-i-gap-row::before { content: "\e05b"; }
     .k-i-handle-resize-alt::before { content: "\e05c"; }
     .k-i-handle-resize::before { content: "\e05d"; }
+    .k-i-shevron-top::before { content: "\e94a"; }
```

Comment:
```suggestion
    .k-i-chevron-double-up::before { content: "\e94a"; }
```

Reply:
Synced with Pavel. He will add the rest three icons and will rename them to `k-i-chevron-double-up|down|left|right`. Once he push them, I'll continue :)